helpers.go

```diff
@@ -132,14 +132,14 @@ func getCassandraType(name string) TypeInfo {
 			Elem:       getCassandraType(strings.TrimPrefix(name[:len(name)-1], "list<")),
 		}
 	} else if strings.HasPrefix(name, "map<") {
-		names := strings.SplitN(strings.TrimPrefix(name[:len(name)-1], "map<"), ", ", 2)
+		names := splitCompositeTypes(strings.TrimPrefix(name[:len(name)-1], "map<"))
```

Comment:
though it shouldn't happen it would be good to make sure there is 2 items here to avoid a seg fault below.

Reply:
What to do then if not panic?

Reply:
I'll print a log at the moment.